far2l/edit.cpp

```diff
@@ -1351,7 +1351,9 @@ int Edit::ProcessKey(int Key)
 			Show();
 			return TRUE;
 		}
-		case KEY_CTRLINS:     case KEY_CTRLNUMPAD0:
+		case KEY_CTRLINS:
```

Comment:
are you sure its needed? There is Key reassignment for KEY_CTRLC: https://github.com/elfmz/far2l/blob/master/far2l/edit.cpp#L695

Reply:
I am not, will check.